pyglmnet/pyglmnet.py

```diff
@@ -731,6 +731,7 @@ def fit(self, X, y):
 
             elif self.solver == 'cdfast':
                 beta_old = deepcopy(beta)
+                z = beta[0] + np.dot(X, beta[1:])
```

Comment:
this seems more like a hack, no? The cache that is computed before is useless now. Could it be fixed instead of recomputing this here?

Reply:
it's recalculated in this block of code since it's invalidated here and it's needed by the call to _cdfast. Since it's the only place that _cdfast is called, though, I'll go ahead and remove `z` from the _cdfast interface entirely.